fluentd/vendored_gem_src/fluent-plugin-viaq_data_model/lib/fluent/plugin/filter_viaq_data_model.rb

```diff
@@ -203,7 +203,12 @@ def configure(conf)
         @docker_hostname = nil
```

Comment:
This change needs to be proposed as a PR to https://github.com/ViaQ/fluent-plugin-viaq_data_model - then we will pull it into fluentd from there